vault/logical_system_paths.go

```diff
@@ -2048,8 +2626,18 @@ func (b *SystemBackend) mountPaths() []*framework.Path {
 		{
 			Pattern: "mounts$",
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation: b.handleMountTable,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: b.handleMountTable,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields:      map[string]*framework.FieldSchema{
+								// TODO not sure how to handle the dynamic key of entry.Path
```

Comment:
I believe with dynamic keys we just added an empty `FieldSchema`

Reply:
removed all occurrences.